### PR TITLE
Add f32 timing variables

### PR DESCRIPTION
**Pull request by Xaeroxe**
This takes a bit of legwork out of a very common task in gamedev.

### CODE REVIEW

**Comment by Xaeroxe** [comment]
cc @jojolepro 

Just wanted to show you we're finally using your function :)

File: src/app.rs [hunk 1]
```diff
@@ -155,8 +155,10 @@ impl<'a, 'b> Application<'a, 'b> {
         {
             let mut time = self.engine.world.write_resource::<Time>();
             time.frame_number += 1;
+            time.delta_seconds = duration_to_secs(self.time.delta_time);
```
Comment (Aceeri) [inline src/app.rs]: I can't say I like that `time.fixed_step` and `time.fixed_seconds` can differ due to human error. Maybe we should put this behind a method to set/get the fields?

**Comment by Xaeroxe** [comment]
@Aceeri This is my best attempt at controlling for mutability.  Let me know what you think.

**Comment by Xaeroxe** [comment]
Squashed.

bors r+

**Pull request merged**